xml/troubleshooting.xml

```diff
@@ -1079,64 +1079,25 @@
   </sect2>
 
   <sect2 xml:id="sec-trouble-login-dk">
-   <title>Login Successful but &gnome; Desktop Fails</title>
+   <title>&gnome; Desktop Has Issues</title>
    <para>
-    If this is the case, it is likely that your &gnome; configuration files
-    have become corrupted. Some symptoms may include the keyboard failing to
-    work, the screen geometry becoming distorted, or even the screen coming up
-    as a bare gray field. The important distinction is that if another user
-    logs in, the machine works normally. It is then likely that the problem can
-    be fixed relatively quickly by simply moving the user's &gnome;
-    configuration directory to a new location, which causes &gnome; to
-    initialize a new one. Although the user is forced to reconfigure &gnome;,
-    no data is lost.
+     If you are experiencing issues with the &gnome; desktop (for example,
+     incorrect screen geometry, blank desktop, etc.), try resetting &gnome; settings.
+     Before you do this, back up the existing configuration files using the following command:
    </para>
-   <procedure xml:id="proc-trouble-login-dk">
```

Comment:
Why did you not keep the procedure structure here? Moving to paras seems like a step back.